protos/pb.proto

```diff
@@ -649,6 +649,8 @@ message ExportRequest {
 	string secret_key = 7;
 	string session_token = 8;
 	bool anonymous = 9;
+
+	uint64 namespace = 10;
```

Comment:
You don't need this. The token will have this information.

Reply:
Check ctx for token, if not found, export default namespace.

How would we export all the namespaces?
 -> Use the guardian of the galaxy token for this.